### ISSUE TITLE
Remember the selected (sub-)folder at menu > My Places > Tracks >

### ISSUE THREAD

**Issue by p-p-s** [osmandapp/Osmand#7672]
Dear OsmAnd team,
I love your wonderful app! Thank you so much!

Still I have a wish. When I visited 
Hamburger menu > My Places > Tracks >
and drilled into the tracks folder structure to pick a track, the next time I visit 
Hamburger menu > My Places >
again in the same session, I would like to find Tracks selected instead of Favorites and the (sub-)folder to be opened where I last time left. 

This would be useful in cases where I picked the wrong track in the first place or did not pick all tracks needed in parallel or one after the other.

**Comment by sonora** [comment]
Yes, something which annoys me daily, as well as other users who view dozens of track-folders using OsmAnd ... :)
